.github/workflows/build.yml

```
@@ -164,6 +164,22 @@ jobs:
           pushd android 1> /dev/null
           ./gradlew clean build check test
         working-directory: example
+      - name: Cache node_modules
+        if: ${{ matrix.os == 'macos-latest' }}
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: rnta-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
```

We should use the same key as elsewhere so we can reuse the same cache:
```suggestion
          key: project-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
```

---

Do we? In other cases we were caching `example/node_modules` based on the `example/yarn.lock` file. Here I am caching the `node_modules` folder from the project root.

---

Yes, we are caching root `node_modules` folder in
https://github.com/microsoft/react-native-test-app/blob/5d75836ccc116d425166b904b56591f1ee1015c7/.github/workflows/build.yml#L21
https://github.com/microsoft/react-native-test-app/blob/5d75836ccc116d425166b904b56591f1ee1015c7/.github/workflows/build.yml#L100
https://github.com/microsoft/react-native-test-app/blob/5d75836ccc116d425166b904b56591f1ee1015c7/.github/workflows/build.yml#L206
https://github.com/microsoft/react-native-test-app/blob/5d75836ccc116d425166b904b56591f1ee1015c7/.github/workflows/build.yml#L274
https://github.com/microsoft/react-native-test-app/blob/5d75836ccc116d425166b904b56591f1ee1015c7/.github/workflows/build.yml#L320